es6/reducers.js

```diff
@@ -2,12 +2,15 @@ export const select = (state) => state.time
 
 const initial_state = {
     speed: 1,
+    warped_time: Date.now(),
```

Comment:
Lets set these both to `undefined` or `null` to begin with, and in `handlerStateChange` only use the value when it's explicitly set to something other than `null`/`undefined`.  That way we don't have to keep these starting values in sync with WarpedTime's.